implementation/gate/src/main/java/eu/efti/eftigate/entity/TransportVehicleEntity.java

```diff
@@ -28,22 +26,13 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @EqualsAndHashCode(callSuper = true)
-@Entity
-@EntityListeners(AuditingEntityListener.class)
-@Table(name = "transportvehicle")
-public class TransportVehicle extends JourneyEntity implements Serializable {
+public class TransportVehicleEntity extends JourneyEntity implements Serializable {
```

Comment:
I would have name it *Dto since it is used as such now, but i guess it will disapear so who care :D